app/templates/settings/apps/unsplash.hbs

```diff
@@ -31,35 +31,36 @@
             <div class="gh-setting-action">
                 <div class="for-checkbox">
                     <label for="isActive" class="checkbox">
+                        {{#gh-form-group errors=model.errors hasValidated=model.hasValidated property="isActive"}}
                         {{one-way-checkbox model.isActive id="isActive" name="isActive" type="checkbox" update=(action "update") data-test-unsplash-checkbox=true}}
```

Comment:
To make the testSelectors a little more manageable, rather than:

```
data-test-unsplash-checkbox=true
```

I've stated using the following pattern:

```
data-test-checkbox="unsplash"
data-test-input="unsplash"
data-test-button="send-request"
```

Then in tests you can do things like:

```
fillIn(testSelector('input', 'unsplash'), 'xxxx')
click(testSelector('button', 'send-request')
```

Not essential but I've found it can make things a bit more consistent 😄